INFO:    Could not find any nv files on this host!
Traceback (most recent call last):
  File "/home/yb1025/.conda/envs/allegro_gym/bin/huggingface-cli", line 8, in <module>
    sys.exit(main())
  File "/home/yb1025/.conda/envs/allegro_gym/lib/python3.6/site-packages/huggingface_hub/commands/huggingface_cli.py", line 41, in main
    service.run()
  File "/home/yb1025/.conda/envs/allegro_gym/lib/python3.6/site-packages/huggingface_hub/commands/user.py", line 169, in run
    username = input("Username: ")
EOFError: EOF when reading a line
Reusing dataset glue (/home/yb1025/.cache/huggingface/datasets/glue/sst2/1.0.0/dacbe3125aa31d7f70367a07a8a9e72a5a0bfeb5fc42e75c9db75b96da6053ad)
  0%|          | 0/3 [00:00<?, ?it/s]100%|##########| 3/3 [00:00<00:00, 30.99it/s]
Loading cached processed dataset at /home/yb1025/.cache/huggingface/datasets/glue/sst2/1.0.0/dacbe3125aa31d7f70367a07a8a9e72a5a0bfeb5fc42e75c9db75b96da6053ad/cache-2baabeaa2f4d4461.arrow
Loading cached processed dataset at /home/yb1025/.cache/huggingface/datasets/glue/sst2/1.0.0/dacbe3125aa31d7f70367a07a8a9e72a5a0bfeb5fc42e75c9db75b96da6053ad/cache-42290a97f21e20f6.arrow
Loading cached processed dataset at /home/yb1025/.cache/huggingface/datasets/glue/sst2/1.0.0/dacbe3125aa31d7f70367a07a8a9e72a5a0bfeb5fc42e75c9db75b96da6053ad/cache-d745c4df4910759d.arrow
Some weights of the model checkpoint at roberta-base were not used when initializing RobertaForSequenceClassification: ['roberta.pooler.dense.weight', 'lm_head.dense.bias', 'roberta.pooler.dense.bias', 'lm_head.layer_norm.bias', 'lm_head.dense.weight', 'lm_head.decoder.weight', 'lm_head.layer_norm.weight', 'lm_head.bias']
- This IS expected if you are initializing RobertaForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing RobertaForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of RobertaForSequenceClassification were not initialized from the model checkpoint at roberta-base and are newly initialized: ['classifier.out_proj.weight', 'classifier.dense.bias', 'classifier.out_proj.bias', 'classifier.dense.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
The following columns in the training set  don't have a corresponding argument in `RobertaForSequenceClassification.forward` and have been ignored: idx, sentence.
***** Running training *****
  Num examples = 67349
  Num Epochs = 5
  Instantaneous batch size per device = 16
  Total train batch size (w. parallel, distributed & accumulation) = 16
  Gradient Accumulation steps = 1
  Total optimization steps = 21050
  0%|          | 0/21050 [00:00<?, ?it/s]  0%|          | 1/21050 [00:04<25:44:52,  4.40s/it]  0%|          | 2/21050 [00:09<26:49:54,  4.59s/it]